### PR TITLE
trim line space

### PR DESCRIPTION
**Pull request by urugang**
requirements refered to #99.

### CODE REVIEW

**Comment by olekukonko** [comment]
@mattn what is your opinion on this?

**Comment by mattn** [comment]
@urugang Could you please add test?

File: table.go [hunk 1]
```diff
@@ -624,12 +624,13 @@ func (t *Table) printRow(columns [][]string, rowIdx int) {
 	}
 	//fmt.Println(max, "\n")
 	for x := 0; x < max; x++ {
+		var buffer strings.Builder
```
Comment (olekukonko) [inline table.go]: Why will you declare `var buffer strings.Builder` in a for loop

Comment (olekukonko) [inline table.go]: Must it be `string.Builder` ?  the `&buffer` in every line does not make this clean

**Comment by olekukonko** [comment]
@mattn kindly close in 30 days, I don't even like the idea of `var` in loops

**Pull request closed without merging**